src/main/java/Model/Database/Interaction/I_Project.java

```diff
@@ -10,10 +10,7 @@
 import Model.Database.Support.SqlConnectionOneTimeReestablisher;
 import Model.Database.Tables.Table.T_Project;
 
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
+import java.sql.*;
```

Comment:
tu je nejaky dovod preco sa importuje vsetko z java.sql.? je to vyhodnejsie alebo to chces kvoli menej importom?